Create README.md (#36)
google-gemini/cookbook: examples/README.md

# Check out the quickstarts folder!

> So far, most of the examples in this cookbook are in the [quickstarts](https://github.com/google-gemini/cookbook/tree/main/quickstarts) folder.

Reviewer: ```suggestion
There are even more examples in the [quickstarts](https://github.com/google-gemini/cookbook/tree/main/quickstarts) folder of this cookbook.
```

Couple nits:
* I think "More examples" conveys "that section is great" rather than "this section is not great".
* "So far" isn't [timeless](https://developers.google.com/style/timeless-documentation)

WDYT?
